src/api/spec/models/configuration_spec.rb

```diff
@@ -33,4 +33,74 @@
       expect(configuration).to have_received(:write_to_backend)
     end
   end
+
+  describe '#ldap_enabled?' do
+    let(:config) { Configuration.first }
+
+    it 'returns true if config option `ldap_mode` is set to :on' do
+      stub_const('CONFIG', CONFIG.merge({ 'ldap_mode' => :on }))
+      expect(config.ldap_enabled?).to eq(true)
+    end
+
+    it 'returns false if config option `ldap_mode` is not set to :on' do
+      stub_const('CONFIG', CONFIG.merge({ 'ldap_mode' => :off }))
+      expect(config.ldap_enabled?).to eq(false)
+    end
+  end
+
+  context '#passwords_changable?' do
+    let(:config) { Configuration.first }
+
+    it 'returns false if config option `change_password` is set to false' do
+      allow(config).to receive(:change_password).and_return(false)
```

Comment:
Discussed offline: Our RSpec set up is not aware of `object.stub[...]` and `allow` is totally fine here.